grammar.y

```diff
@@ -776,7 +775,6 @@ expr:
     $$ = &ConvertExpr{Expr: $3, Type: $5}
   }
 | function_call_keyword
-| function_call_custom
```

Comment:
I'm relying on the `function_call_generic` grammar rule that already existed for the custom functions